dev/components/dancing-dots.html

```diff
@@ -77,21 +67,31 @@
       this._active ? this._start() : this._stop();
     }
     _start() {
-      if (!this._interval) {
-        this.disabled = false;
-        this.setAttribute('animate', '');
-        let animate = () => {
-          this.setAttribute('up', '');
-          setTimeout(() => this.removeAttribute('up'), 280);
+      this.querySelectorAll('bullet').forEach((bullet, i) => {
+        let keyframes = {transform: ['translateY(8px)', 'translateY(-8px)']};
+        let timing = {
+          iterations: Infinity,
+          duration: 200,
+          direction: 'alternate',
+          iterationStart: 0.5,
+          easing: 'ease-in-out',
+          delay: i * 40,
         };
-        this._interval = setInterval(animate, 560);
-      }
+        bullet._animation = bullet.animate(keyframes, timing);
+      });
     }
     _stop() {
-      this.removeAttribute('animate');
-      if (this._interval) {
-        clearInterval(this._interval);
-        this._interval = null;
+      for (let bullet of this.querySelectorAll('bullet')) {
```

Comment:
Could use a helper method like `_bulletElements` that returns `this.querySelectorAll('bullet')` here and above to share the string constant and querySelectorAll call.